pyccel/commands/lambdify.py

```diff
@@ -5,8 +5,9 @@
 import sympy as sp
 from packaging import version
 
-from pyccel.commands.epyccel import epyccel
+from pyccel.commands.epyccel  import epyccel
```

Comment:
I think that this change is unnecessary

Reply:
I agree it is purely aesthetic. Yaman likes things to be lined up so with this change we have:
```python
from pyccel.commands.epyccel  import epyccel
from pyccel.utilities.strings import random_string
from pyccel.errors.errors     import PyccelError
```
instead of:
```python
from pyccel.commands.epyccel import epyccel
from pyccel.utilities.strings import random_string
from pyccel.errors.errors import PyccelError
```

Reply:
😅